eng/pipelines/build-unix-job.yml

```diff
@@ -0,0 +1,46 @@
+# Build on Unix
+parameters:
+- name: jobName
+  type: string
+  default: ''
+- name: testArtifactName
+  type: string
+  default: ''
+- name: configuration
+  type: string
+  default: 'Debug'
+- name: queueName
+  type: string
+  default: ''
+- name: vmImageName
+  type: string
+  default: ''
+
+jobs:
+- job: ${{ parameters.jobName }}
+  pool:
+    ${{ if ne(parameters.queueName, '') }}:
+      name: NetCorePublic-Pool
+      queue: ${{ parameters.queueName }}
```

Comment:
I considered this but chose to add the `Name` suffix because I wanted to make it easier to grep for these and keep it visible separate from the property name.